libs/SalesforceSDK/src/com/salesforce/androidsdk/rest/ClientManager.java

```diff
@@ -113,7 +113,7 @@ public void getRestClient(Activity activityContext, RestClientCallback restClien
             if (options != null) {
                 i.putExtras(options);
             }
-            activityContext.startActivity(i);
+            activityContext.startActivityForResult(i, 0);
```

Comment:
I'm not certain we'd merge it this way, but this one line allows me to pause the app at the login activity, resume and still be able to complete login since the system keeps the activity with the broadcast intent receivers while awaiting the fictitious result.

Reply:
I'm okay with that approach for the bug fix (maybe with a big comment to explain the reason for using startActivityForResult). We can try and figure out if there is a more elegant approach post 11.0.
@brandonpage what do you think?

Reply:
That's a thought, since we could do this short term and maybe a heftier effort post 11.0.  For this short-term fix, I suspect this same change would need to be applied to the other locations where `LoginActivity` is started.

Reply:
This seems fine for an 11.0 bugfix.  Ditto on the explanation comment, maybe even with a todo.  Now that we see the underlying problem I think any other fix would be a bit too risky so close to release.

Reply:
Thanks for the feedback, all.  I did a lot of reading and testing today on emulators and my devices.  Before switching to `startActivityForResult`, the bug is really easy to consistently reproduce 100% of the time.  Afterwards, it seems to consistently behave as expected.

One test I did a lot was send the app to background and run enough memory-heavy video games (my kids!) to force the app to terminate and resume from the task's base intent when using the recent tasks list.  (Note: Minecraft is the memory-use winner).  The base intent when using `startActivityForResult` is able to create a new task and activity stack that will let the user log in.

I _did_ try to find another Android API that would create the same sub-activity relationship without the "result" semantic this technically doesn't use.  I couldn't find a better match though at the activity level.  I'll add a comment to explain this and we can look for a solution to the underlying issue in the next release.

Reply:
I should mention: I _did_ look at other parts of the app that start login activity.  This one spot seems the one we can apply this work-around to at the moment within the MSDK code.  If anyone thinks there's another, let me know how to test it.